packages/host/app/components/operator-mode/stack.gts

```diff
@@ -37,6 +37,17 @@ export default class OperatorModeStack extends Component<Signature> {
     for (let i = this.args.stackItems.length - 1; i > itemIndex; i--) {
       itemsToDismiss.push(this.args.stackItems[i]);
     }
+
+    // add closing animation on last item
+    const lastItemIndex = this.args.stackItems.length - 1;
+    const lastItemEl = document.querySelector(
+      `[data-stack-card-index="${lastItemIndex}"]`,
+    ) as HTMLElement;
```

Comment:
We should not be calling `querySelector` outside of our tests. An ember modifer is a much better choice here